outreach-communication.qmd

```diff
@@ -0,0 +1,251 @@
+
+-   Leave Sender name and Sender email as is
+
+-   Hit "Send now" button to send email
+
+## Group Emails
+
+We have two shared emails, `cct-datascience@arizona.edu` and `cct-datascience-admins@arizona.edu`.
+These should go to all people in the group.
+When you respond, it will be from your address but please keep the original group email address in cc as long as is useful.
+
+-   `cct-datascience@arizona.edu` is the general 'how to contact us', for use in websites, business cards, posters, etc, in many cases alongside a personal email address.
+
+-   `cct-datascience-admins@arizona.edu` is for shared accounts, licenses, billing etc.
+
+## Group Website UA quickstart (Drupal)
+
+Website [here](https://datascience.cals.arizona.edu/)
+
+Links to documentation:
+
+-   **Complete Quickstart Docs** instructions for how to change content or appearance of a UA website: <https://uasites.arizona.edu/home>
+
+-   **Sign In**
+
+    1.  Go to <https://datascience.cct.arizona.edu/user/>
+
+    2.  Click on the "Log in using UA WebAuth" button, click on "LOG IN", and then sign in with UA credentials
+
+    3.  You can [add new content](https://uasites.arizona.edu/node/11_) or [edit existing content](https://uasites.arizona.edu/node/3)
+
+-   **People**
+
+    -   **Move people around** <https://datascience.cct.arizona.edu/admin/uaqs-person/reorder-people>
+
+    -   **Add / edit people** <https://quickstart.arizona.edu/people-directories>
+
+-   **Add blogpost** (this is added as a 'news' type) <https://quickstart.arizona.edu/news-content-type>
+
+-   **Update Menu** <https://quickstart.arizona.edu/setting-main-menu>
+
+-   **Preview page without publishing**
+
+    -   Save the page unpublished so that you can see how it will display without the public seeing it.
+
+-   **Clear cache** so it appears to public immediately
+
+    -   [Configuration --\> Development --\> Performance](https://datascience.cct.arizona.edu/admin/config/development/performance) --\> Clear all caches
+
+-   Projects
+
+    -   Card summary - 1-2 sentences, only appears on the card
+
+## Social Media
+
+### Twitter
+
+We no longer post to our Twitter/X account [\@cct_datascience](https://twitter.com/cct_datascience)
+
+### Mastodon / fediverse
+
+We have a mastodon account [\@cct_datascience\@fosstodon.org](https://fosstodon.org/@cct_datascience).
+To log in, you'll find the password on [stache](https://stache.arizona.edu/) (you may need to ask for it to be shared with you if you don't see it)
+
+#### Guidelines
+
+-   Please review the group [code of conduct](code-of-conduct.qmd) before posting to ensure you adhere to these guidelines
+
+-   Including relevant pictures in posts are encouraged.
+    **Please get explicit verbal/written consent from everyone in the picture before posting.**
+
+-   Cite sources and mention them (\@journal or \@author) if possible
+
+-   If anyone comments on a post, make sure to respond to them, especially if it's a question.
```

Comment:
Perhaps add `Responses can include emojis, comments, and replies. If responding out of channel, consider replying with an emoji in channel`?

Reply:
I didn't quite understand this comment.  I'm going to go ahead and merge this PR, but please feel free to open an issue with more details on this.